tasks/build.js

```diff
@@ -29,33 +30,39 @@ function run(task, opts) {
 	return new Promise(function(resolve, reject) {
 		if (opts.watch) {
 			console.log("Watching " + getGlob(task) + " and will trigger " + task);
-			gulp.watch.call(gulp, getGlob(task), [task])
-				.on('end', resolve)
-				.on('error', reject);
+			gulp.watch(getGlob(task), [task]);
 		} else {
-			gulp.start.call(gulp, [task])
-				.on('end', resolve)
-				.on('error', reject);
+			gulp.start([task], resolve);
 		}
 	});
 }
 
 gulp.task('build-sass', function() {
 	return obt.build.sass(gulp, {
-		sass: sourceFolder + mainScssFile,
-		buildFolder: buildFolder,
-		sourcemaps: true,
-		env: 'production'
-	});
+			sass: sourceFolder + mainScssFile,
+			buildFolder: buildFolder,
+			env: yargs.argv.dev ? 'development' : 'production'
```

Comment:
yeah, passed in https://github.com/Financial-Times/next-build-tools/blob/correct-dev-mode/tasks/build.js#L82, but args can't be passed to tasks. guess it could be made global at that point, but seemed messy. might be safer though